core/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 # VecMem project, part of the ACTS project (R&D line)
 #
-# (c) 2021-2024 CERN for the benefit of the ACTS project
+# (c) 2021-2023 CERN for the benefit of the ACTS project
```

Comment:
Actually, changing this year back to 2023 makes no sense at all. Should remain at 2024.

Reply:
:confused: I used GitHub's built-in PR reversal feature. That undoes everything from the previous PR.